ykcompile/src/lib.rs

```diff
@@ -975,7 +975,8 @@ impl<TT> TraceCompiler<TT> {
             Location::NotLive => unreachable!(),
         }
         let dest_loc = self.iplace_to_location(dest);
-        self.store_raw(&dest_loc, &*TEMP_LOC, SIR.ty(&src.ty()).size());
+        debug_assert_eq!(SIR.ty(&dest.ty()).size(), *PTR_SIZE);
+        self.store_raw(&dest_loc, &*TEMP_LOC, *PTR_SIZE);
```

Comment:
This was actually a bug. Might it be worth putting it in a separate commit with an explanation?

Reply:
Can do, but will have to force push.